src/daffodilDebugger.ts

```diff
@@ -104,42 +101,25 @@ export async function getDebugger(config: vscode.DebugConfiguration) {
 
       // Code for downloading and setting up daffodil-debugger files
       if (!fs.existsSync(`${rootPath}/${artifact.name}`)) {
```

Comment:
No because `${artifact.name}` looks like `daffodil-debugger-${this.daffodilVersion}-${this.version}` at the top of the file. so if you have two different version such as `daffodil-debugger-3.1.0-1.0.0` and `daffodil-debugger-3.1.0-1.0.1` which will be there own specific folders. All you would need to make sure is that the version you want to test is updated in `package.json` and then run `yarn compile` to ensure `src/version.ts` has the same version as `package.json` and you will be good to go.